apps/els_lsp/test/els_parser_SUITE.erl

```diff
@@ -104,6 +105,28 @@ parse_incomplete_spec(_Config) ->
   ?assertMatch([#{id := aa}], parse_find_pois(Text, atom)),
   ok.
 
+%% Issue #1171
+parse_no_tokens(_Config) ->
+  %% scanning text containing only whitespaces returns an empty list of tokens,
+  %% which used to crash els_parser
+  Text1 = "  \n  ",
+  {ok, []} = els_parser:parse(Text1),
+  %% `els_parser:parse' actually catches the exception and only prints a warning
+  %% log. In order to make sure there is no crash, we need to call an internal
+  %% debug function that would really crash and make the test case fail
+  error = els_parser:parse_incomplete_text(Text1, {1, 1}),
```

Comment:
I wonder if there is a better way to make sure no warning log is emitted in ct tests?

Reply:
Not sure I'm following 100%, but an option could be to (eventually isolate and) mock with passthrough the function generating the warning and check from CT if the function was called via `meck_history` or similar.

Reply:
good idea